### PR TITLE
replace firewalld with iptables.

### PR DESCRIPTION
**Pull request by nirajdp76**
For CENTOS/RHEL leverage iptables. this will also ensure firewalld is shutdown and disabled.

1: openstack-firewall role, it only has task/main.yml .. very similar to openstack-database role. This will remove all the ufw and firewalld from each individual openstack project role and they pass the rule creation information to openstack-firewall from their role/meta
2: For RHEL/CENTOS persist iptables rule in config file and use iptables ansible module to apply at runtime.. this ill take away all the restart iptables calls.

Note: Ported anisble iptables module from ansible 2.2 that included bug fixes as well as feature add which allowed passing action (insert or append) which is needed to ensure rules were written before reject.

### CODE REVIEW

**Comment by nirajdp76** [comment]
retest

File: roles/aodh/tasks/main.yml [hunk 1]
```diff
@@ -13,33 +13,20 @@
...
-      cmd: "{{ item.cmd }}"
-      config_dirs: "{{ item.config_dirs }}"
-    with_items:
-      - "{{ aodh.services.aodh_api}}"
-      - "{{ aodh.services.aodh_evaluator }}"
-      - "{{ aodh.services.aodh_listener }}"
-      - "{{ aodh.services.aodh_notifier }}"
+- name: install aodh services
+  upstart_service:
+    name: "{{ item.nam }}"
```
Comment (ddaskal) [inline roles/aodh/tasks/main.yml]: typo, should be `item.name`

File: roles/ceilometer-control/tasks/main.yml [hunk 2]
```diff
@@ -1,45 +1,31 @@
...
-      cmd: "{{ item.cmd }}"
-    with_items:
-      - "{{ ceilometer.services.ceilometer_api }}"
-      - "{{ ceilometer.services.ceilometer_collector }}"
-      - "{{ ceilometer.services.ceilometer_polling }}"
-      - "{{ ceilometer.services.ceilometer_notification }}"
-    notify: restart ceilometer services
+- name: install ceilometer controller services
+  upstart_service:
+    name: "{{ item.nam }}"
```
Comment (ddaskal) [inline roles/ceilometer-control/tasks/main.yml]: typo, should be `item.name`

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
Test output: https://gist.github.com/nirajdp76/895710cb63a7fb1efdac609b1c72068e

**Comment by nirajdp76** [comment]
retest

**Comment by wangxian-ibm** [comment]
PR tested, no issues were seeing from networking perspective. 👍

**Pull request merged**